compiler/src/Language/Bond/Codegen/TypeMapping.hs

```diff
@@ -17,7 +17,7 @@ type system of a target programming language.
 module Language.Bond.Codegen.TypeMapping
     ( -- * Mapping context
       MappingContext(..)
-    , TypeMapping
+    , TypeMapping(..)
```

Comment:
`TypeMapping` also refers to `TypeNameBuilder`. While we don't need to export that from the module because its just a type alias, doing so would allow you to use `TypeNameBuilder` instead of `Reader MappingContext Builder`.